arangod/RestServer/FlushFeature.cpp

```diff
@@ -161,6 +163,13 @@ void FlushFeature::executeCallbacks() {
     }
     // TODO: honor the commit results here
```

Comment:
I believe we can implement more fine-grained approach, something like within a loop:
```
arangodb::aql::QueryCache::instance()->invalidate(trx.name());
```